views/login/sin.pug

```diff
@@ -15,16 +15,16 @@ block content
     div
       label(for='sin') Social Insurance Number
       span.pure-form-message For Example: 123 456 789
-      input.w-3-4#sin(class={['has-error']: errors && errors.code} name='sin', type='text', value=data.code, autocomplete='off' aria-describedby=(errors && errors.code ? 'code_error' : false))
+      input.w-3-4#sin(class={['has-error']: errors && errors.code} name='sin', type='text', value=data.code, autocomplete='off' aria-describedby=(errors && errors.sin ? 'sin_error' : false))
```

Comment:
- `errors.code` should be `errors.sin`
- `value=data.code` should be `value=data.sin`